studio/components/interfaces/Reports/renderers/ApiRenderers.tsx

```diff
@@ -105,42 +107,89 @@ export const TopApiRoutesRenderer = (
-                    {Number(datum.avg).toFixed(2)}ms
-                  </Table.td>
-                )}
-              </Table.tr>
-            </Fragment>
-          ))}
-        </>
-      }
-    />
+    <Collapsible>
+      <Table
+        head={
+          <>
+            <Table.th className={headerClasses}>Request</Table.th>
+            <Table.th className={headerClasses + ' text-right'}>Count</Table.th>
+            {props.data[0].avg !== undefined && (
+              <Table.th className={headerClasses + ' text-right'}>Avg</Table.th>
+            )}
+          </>
+        }
+        body={
+          <>
+            {props.data.map((datum, index) => (
+              <Fragment key={datum.path + (datum.search || '')}>
+                <Table.tr
+                  className={[
+                    'p-0 transition transform duration-700',
+                    showMore && index >= 3 ? 'w-full h-full opacity-100' : '',
+                    !showMore && index >= 3 ? ' w-0 h-0 translate-y-10 opacity-0' : '',
+                  ].join(' ')}
+                >
+                  {(!showMore && index < 3) || showMore ? (
+                    <>
+                      <Table.td className={[cellClasses].join(' ')}>
+                        <RouteTdContent {...datum} />
+                      </Table.td>
+                      <Table.td className={[cellClasses, 'text-right align-top'].join(' ')}>
+                        {datum.count}
+                      </Table.td>
+                      {props.data[0].avg !== undefined && (
+                        <Table.td className={[cellClasses, 'text-right align-top'].join(' ')}>
+                          {Number(datum.avg).toFixed(2)}ms
+                        </Table.td>
+                      )}
+                    </>
+                  ) : null}
+                </Table.tr>
+              </Fragment>
+            ))}
+          </>
+        }
+      />
+      <Collapsible.Trigger asChild>
+        <div className="flex flex-row justify-end w-full gap-2 pt-1">
+          <Button
+            type="text"
+            onClick={() => setShowMore(!showMore)}
+            className={[
+              'transition',
+              showMore ? 'text-foreground' : 'text-foreground-lighter',
+              props.data.length <= 3 ? 'hidden' : '',
+            ].join(' ')}
+          >
+            {!showMore ? 'Show more' : 'Show less'}
+          </Button>
+          <Button
+            type="text"
+            className="text-foreground-lighter"
+            onClick={() => {
+              props.router.push({
+                pathname: `/project/${projectRef}/logs/explorer`,
+                query: {
+                  q: props.params?.sql,
+                  its: props.params!.iso_timestamp_start,
+                  ite: props.params!.iso_timestamp_end,
+                },
+              })
+            }}
+          >
+            Open in Logs Explorer
+          </Button>
```

Comment:
Small styling nit:
<img width="227" alt="Screenshot 2023-10-02 at 14 53 12" src="https://github.com/supabase/supabase/assets/10985857/09d06587-d1c9-43a2-a859-f63019f44818">
the padding around this button looks incosistent

Reply:
yup - a little more obvious in dark mode 
![image](https://github.com/supabase/supabase/assets/19742402/7145284c-9585-4cf4-b5ab-dfe8ef42a7eb)